1_C++/suffix_count/build/readme.md

```diff
@@ -0,0 +1,4 @@
+# The build folder
```

Comment:
Do we need another readme? I think we can state this in the main readme. - The less files the better

Reply:
You can't commit empty folders to Git, and the build will fail if the build folder doesn't exist, so I used this as a way to make it so that it would be present in the template. An alternative would be conditionally creating the build/ folder in the compile script. What are your thoughts?

Reply:
this is fine then. no changes